service/worker_slice_gateway_service.go

```diff
@@ -627,7 +624,7 @@ func (s *WorkerSliceGatewayService) generateCerts(ctx context.Context, sliceName
 	environment["SLICE_NAME"] = sliceName
 	environment["CERT_GEN_REQUESTS"], _ = util.EncodeToBase64(&cpr)
 	util.CtxLogger(ctx).Info("jobNamespace", jobNamespace) //todo:remove
-	_, err := s.js.CreateJob(ctx, jobNamespace, JobImage, environment)
+	_, err := s.js.CreateJob(ctx, os.Getenv("KUBESLICE_CONTROLLER_MANAGER_NAMESPACE"), JobImage, environment)
```

Comment:
can you please take care of 	util.CtxLogger(ctx).Info("jobNamespace", jobNamespace) //todo:remove